scraping/x/model.py

```diff
@@ -27,13 +27,20 @@ class XContent(BaseModel):
     )
 
     @classmethod
-    def to_data_entity(cls, content: "XContent") -> DataEntity:
+    def to_data_entity(
+        cls, content: "XContent", obfuscate_content_date: bool
+    ) -> DataEntity:
         """Converts the XContent to a DataEntity."""
+        entity_timestamp = content.timestamp
+        if obfuscate_content_date:
+            content.timestamp = entity_timestamp.replace(
```

Comment:
nit: Consider putting this into a util so it's easy to see we're consistent across all DataSources

Reply:
Done.